motionplan/tpspace/ptgDiffDrive.go

```diff
@@ -0,0 +1,74 @@
+package tpspace
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/golang/geo/r3"
+
+	"go.viam.com/rdk/referenceframe"
+	"go.viam.com/rdk/spatialmath"
+)
+
+// ptgDiffDrive defines a PTG family composed of circular trajectories with an alpha-dependent radius.
```

Comment:
I'm confused by this comment, since I thought this PTG was for move straight and spin, instead of circular trajectories.

Reply:
Same as above; missed this comment.